.github/workflows/ci.yml

```diff
@@ -131,12 +131,6 @@ jobs:
                 if test "$DOWNSTREAM_PROJECT" = "mirgecom"; then
                     # Test main branch
                     examples/run_examples.sh ./examples
-                    
-                    # Test production branch
-                    curl -O -L https://raw.githubusercontent.com/illinois-ceesd/emirge/main/switch-mirgecom.sh
-                    chmod a+x switch-mirgecom.sh
-                    ./switch-mirgecom.sh prod
-                    examples/run_examples.sh ./examples
```

Comment:
```suggestion
                    
                    # Test production drivers
                    . .ci_support/production_testing_env.sh
                    . .ci_support/merge-install-production-branch.sh .
                    . .ci_support/production-drivers-install.sh .
                    . .ci_support/production-drivers-run.sh .
```

Reply:
This should run the production tests  kind of in the same sort of way as the regular examples are exercised.   The big difference here is that the "production" branch will be merged first.

Reply:
Added in #224.